docs/examples/production/rabbitmq.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: production
+spec:
+  replicas: 3
+  resources:
+    requests:
+      cpu: 2
+      memory: 8Gi
+    limits:
+      cpu: 2
+      memory: 8Gi
+  rabbitmq:
+    additionalConfig: |
+      cluster_partition_handling = ignore
+      vm_memory_high_watermark_paging_ratio = 0.99
+      disk_free_limit.relative = 1.0
+  persistence:
+    storageClassName: ssd
+    storage: "128Gi"
+  override:
+    statefulSet:
+      spec:
+        template:
+          spec:
+            containers: []
+            topologySpreadConstraints:
+            - maxSkew: 1
+              topologyKey: zone
+              whenUnsatisfiable: DoNotSchedule
+              labelSelector:
+                matchLabels:
+                  app.kubernetes.io/component: rabbitmq
```

Comment:
I think the labels here should be an instance specific label. You can use this one `https://github.com/rabbitmq/cluster-operator/blob/main/internal/metadata/label.go#L20`

Reply:
I wanted to use this because instance-specific name would need to be adjusted whenever someone wants to use this example (otherwise we would need templating or YAML anchors). Are you sure it will not work like that? I'd be surprised if settings of one instance could affect how pods of another instance are scheduled.

Reply:
You are right! I think it will work with this.